modules/store/src/helpers.ts

```diff
@@ -5,3 +5,12 @@ export function capitalize<T extends string>(text: T): Capitalize<T> {
 export function uncapitalize<T extends string>(text: T): Uncapitalize<T> {
   return (text.charAt(0).toLowerCase() + text.substring(1)) as Uncapitalize<T>;
 }
+
+export function assertNotUndefined<T>(
+  value: T | undefined,
+  message = `${value} must not be undefined`
+): asserts value is T {
+  if (value === undefined) {
+    throw new Error(message);
+  }
+}
```

Comment:
The default message will throw the following error: `undefined must not be undefined`. For better DX, we can provide a more descriptive error message. I also renamed the function to avoid double negation: `assertNotUndefined` => `assertDefined`:

```suggestion
export function assertDefined<T>(
  value: T | null | undefined,
  name: string
): asserts value is T {
  if (value == null) {
    throw new Error(`${name} must be defined.`);
  }
}
```

It's called from the store as follows:

```ts
assertDefined(this.injector, 'Store Injector');
```

And throws the `Store Injector must be defined.` error.